feat: Post announcing Contra sponsorship (#889)
eslint/website: _posts/2021-12-01-contra-gold-sponsor-eslint.md

---
title: "Contra becomes ESLint gold sponsor"

> tags:

Author: ```suggestion
categories:
```

  - Sponsorships
authors:
  - nzakas
---

<p class="text-center">

>     <a href="https://contra.com" title="Nx" rel="noopener nofollow" target="_blank"><img class="lazyload" width="170" data-src="/assets/img/logos/contra.jpg" alt="Contra" src="/assets/img/logos/contra.jpg"></a>

Reviewer: "title" should say Contra

>     <a href="https://contra.com" title="Nx" rel="noopener nofollow" target="_blank"><img class="lazyload" width="170" data-src="/assets/img/logos/contra.jpg" alt="Contra" src="/assets/img/logos/contra.jpg"></a>

Author: ```suggestion
    <a href="https://contra.com" title="Contra" rel="noopener nofollow" target="_blank"><img class="lazyload" width="170" data-src="/assets/img/logos/contra.jpg" alt="Contra" src="/assets/img/logos/contra.jpg"></a>
```

    <a href="https://eslint.org/" title="ESLint" target="_blank"><img class="lazyload" width="200" data-src="/assets/img/logo.svg" alt="ESLint" src="/assets/img/logo.svg"></a>
</p>

> We are happy to share that [Contra](https://nx.dev/) has become an ESLint gold sponsor, donating $1,000 each month for the ongoing maintenance and development of ESLint! Contra is a professional network built around independent work, allowing designers, developers, and more, to live and work the way they want.

Reviewer: Link should be https://contra.com/

> We are happy to share that [Contra](https://nx.dev/) has become an ESLint gold sponsor, donating $1,000 each month for the ongoing maintenance and development of ESLint! Contra is a professional network built around independent work, allowing designers, developers, and more, to live and work the way they want.

Author: ```suggestion
We are happy to share that [Contra](https://contra.com/) has become an ESLint gold sponsor, donating $1,000 each month for the ongoing maintenance and development of ESLint! Contra is a professional network built around independent work, allowing designers, developers, and more, to live and work the way they want.
```

Here's what the folks at Contra had to say about ESLint:

> "ESLint plays a major role in helping us create a safe and consistent experience across a growing number of codebases and engineers. We are grateful for all the open-source projects that are helping us build Contra, and ESLint is the first of many open-source projects that we are committed to back."
> -– Gajus Kuizinas, Co-Founder / CTO at Contra

> ESLint is grateful for the support of the Contra team and we are putting these funds to work by paying more contributors to help maintain and develop ESLint. Donations such as these have allowed us to take on more ambitious projects, such as redesigning the ESLint website, with more to come in 2022.

Reviewer 2: ```suggestion
ESLint is grateful for the support of the Contra team, and we are putting these funds to work by paying more contributors to help maintain and develop ESLint. Donations such as these have allowed us to take on more ambitious projects, such as redesigning the ESLint website, with more to come in 2022.
```

Not critical by any means, but it's a compound sentence and reads a little more nicely (imo) with the comma.
